### ISSUE TITLE
Error compiling when importing sub asset from external module

### ISSUE THREAD

**Issue by Xli33** [yellowspot/vite-plugin-externals#1]
this is the simple vite plugins config
```
plugins: [
      vue(),
      externals({
        vue: 'Vue',
        'vue-router': 'VueRouter',
        axios: 'axios',
        'view-ui-plus': 'ViewUIPlus'
        // pinia: 'Pinia',
        // 'vue-demi': 'VueDemi'
      }),
]
```
and i wanna import the locales from 'view-ui-plus' at node_module, so it's like this

` import msgs from 'view-ui-plus/dist/locale/zh-CN' `

however the import ***view-ui-plus/dist/locale/zh-CN*** is compiled to

**`http://localhost:8000/node_modules/.@yellowspot_vite-plugin-external/view-ui-plus.cjs/dist/locale/zh-CN`**

it's because the `view-ui-plus` is replaced with `node_modules/.@yellowspot_vite-plugin-external/view-ui-plus.cjs`

there's a reproduction [here](https://stackblitz.com/edit/vitejs-vite-7jrfbj?file=vite.config.js,index.html,main.js&terminal=dev)
![image](https://user-images.githubusercontent.com/28592778/235584464-98da5ae8-c7bd-432b-a2a4-5a8306ae84ae.png)

┭┮﹏┭┮

**Comment by mgrunberg** [comment]
@Xli33 thanks for reporting the issue. I released v0.2.0. It solves the issue.

**Comment by Xli33** [comment]
> @Xli33 thanks for reporting the issue. I released v0.2.0. It solves the issue.

thanks for your great work ♪(･ω･)ﾉ
